server/Configurations/googleAuth/passportGoogleConfig.js

```diff
@@ -0,0 +1,43 @@
+require("dotenv").config();
+var GoogleStrategy = require("passport-google-oauth").OAuth2Strategy;
+const db = require("../../database/configurationSequelize");
+ const {findUserOf} = require ('../../routes/user')
+ const User = db.account;
+module.exports = (passport) => {
+  passport.use(
+    new GoogleStrategy(
+      {
+        clientID: process.env.GOOGLE_AUTH_CLIENT_ID,
+        clientSecret: process.env.GOOGLE_AUTH_CLIENT_SECRET_KEY,
+        callbackURL: "http://localhost:5000/auth/google/callback",
+      },
+
+      (token, refreshToken, profile, done) => {
```

Comment:
I'm not sure where the ```refreshToken``` declared in the callback of GoogleStrategy is used? Is this something that happens internally when the user authenticates?